app/Http/Controllers/Admin/ProductCrudController.php

```diff
@@ -206,11 +207,91 @@ protected function setupCreateOperation()
             ],
         ]);
 
+        CRUD::field('main_image')
+                ->label('Main Image')
+                ->type('image')
+                ->tab('Media')
+                ->wrapper(['class' => 'form-group col-md-4'])
+                ->withMedia();
+
+        CRUD::field('privacy_policy')
+                ->label('Privacy policy document')
+                ->type('upload')
+                ->tab('Media')
+                ->wrapper(['class' => 'form-group col-md-4'])
+                ->withMedia();
+
+        CRUD::field('specifications')
+                ->label('Specifications')
+                ->type('upload_multiple')
+                ->tab('Media')
+                ->wrapper(['class' => 'form-group col-md-4'])
+                ->withMedia();
+
+        CRUD::field('gallery')
+            ->type('repeatable')
+            ->tab('Media')
+            ->subfields([
+                [
+                    'name'    => 'image_title',
+                    'type'    => 'text',
+                    'wrapper' => [
+                        'class' => 'form-group col-md-6',
+                    ],
+                ],
+                [
+                    'name'    => 'gallery_image',
+                    'label'   => 'image',
+                    'type'    => 'image',
+                    'wrapper' => [
+                        'class' => 'form-group col-md-6',
+                    ],
+                    'withMedia' => true,
+                ],
+
+                [
+                    'name'    => 'gallery_image_drm',
+                    'label'   => 'Image DRM',
+                    'type'    => 'upload',
+                    'wrapper' => [
+                        'class' => 'form-group col-md-6',
+                    ],
+                    'withMedia' => true,
+                ],
+                [
+                    'name'    => 'gallery_image_specifications',
+                    'label'   => 'Image Specifications',
+                    'type'    => 'upload_multiple',
+                    'wrapper' => [
+                        'class' => 'form-group col-md-6',
+                    ],
+                    'withMedia' => true,
+                ],
+                [
+                    'name'    => 'gallery_image_certificates',
+                    'label'   => 'Image Certificates',
+                    'type'    => 'dropzone',
+                    'wrapper' => [
+                        'class' => 'form-group col-md-6',
+                    ],
+                    'withMedia' => true,
+                ],
+            ]);
+
         $this->crud->setOperationSetting('contentClass', 'col-md-12');
     }
 
     protected function setupUpdateOperation()
     {
         $this->setupCreateOperation();
     }
+
+    public function dropzoneUpload()
+    {
+        if (app('env') === 'production') {
+            return response()->json([]);
+        }
+
+        return $this->traitDropzoneUpload();
+    }
```

Comment:
Not urgent - just a thought. Right now I just want to merge this ASAP.

Reply:
Are you thinking something like this:
```php
if (! env('APP_ENV') === 'production') {
    trait DropzoneOperation
    {
        public function dropzoneUpload()
        {
            return response()->json([]);
        }
    }
} else {
    trait DropzoneOperation
    {
        use \Backpack\Pro\Http\Controllers\Operations\DropzoneOperation;
    }
}
```

And then in demo we use `App\Http\Controllers\Operations\DropzoneOperation` instead of the one in the package ? 

If you agree I can push this change. 

Cheers